build/WinObjC.Frameworks/WinObjC.Frameworks.nuproj

```diff
@@ -110,6 +110,9 @@
     <ProjectReference Include="$(MSBuildThisFileDirectory)..\UIKit.Xaml\dll\UIKit.Xaml.vcxproj" />
     <ProjectReference Include="$(MSBuildThisFileDirectory)..\UIKit\dll\UIKit.vcxproj" />
     <ProjectReference Include="$(MSBuildThisFileDirectory)..\WebKit\dll\WebKit.vcxproj" />
+
+  <!-- NuProj Project References. Ensure that these only get evaluated once by checking that this isn't the second time through this project -->
+  <ItemGroup Condition="'$(SkipGetPackageContentsForOtherPlatforms)' != 'true'">
```

Comment:
Correct. It gets set [here](https://github.com/Microsoft/WinObjC/blob/develop/common/winobjc.nuproj.common.targets#L17) which is when the .nuproj gets re-evaluated for the other configs.